pkg/api/v1/types.go

```diff
@@ -2312,6 +2323,9 @@ type DeleteOptions struct {
 	// Must be fulfilled before a deletion is carried out. If not possible, a 409 Conflict status will be
 	// returned.
 	Preconditions *Preconditions `json:"preconditions,omitempty"`
+
+	// Should the dependent objects be orphaned.
```

Comment:
And the way this is implemented is by adding the "orphan" finalizer, correct? I think this is worth stating in the comment.